packages/cli/src/commands/setup/vite/viteHandler.js

```diff
@@ -75,8 +82,9 @@ export const handler = async ({ force, verbose, addPackage }) => {
         },
       },
       {
-        ...addWebPackages([`@redwoodjs/vite@${version}`]),
-        title: 'Adding @redwoodjs/vite dependency...',
+        // @NOTE: make sure its added as a dev package.
```

Comment:
This step will fail if `@redwoodjs/vite` is already a prod dependency (i.e. if you set it up before)